mp/src/game/server/neo/neo_grenade.h

```diff
@@ -26,6 +26,7 @@ class CNEOGrenadeFrag : public CBaseGrenade
 
 public:
 	virtual void	Explode(trace_t* pTrace, int bitsDamageType);
+	virtual	void	Detonate(void);
```

Comment:
Nit: should mark as `override`. Also, the `(void)` is redundant for modern C++. So preferrably: `virtual void Detonate() override`